vcf/queryInfo.go

```diff
@@ -81,6 +81,10 @@ func parseValue(v Vcf, s []string, k Key) interface{} {
 		for i := range s {
 			stringValues = strings.Split(s[i], ",")
 			for j := range stringValues {
+				if stringValues[j] == "." {
```

Comment:
Can you point me in the direction of where I can see an example of this input s []string?